apps/docs/pages/guides/getting-started/tutorials/with-kotlin.mdx

```diff
@@ -0,0 +1,816 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Build a Product Management App with Jetpack Compose in Android',
+  description: 'Learn how to use Supabase in your Android Kotlin App.',
+}
+
+<QuickstartIntro />
```

Comment:
The quickstart intro mentions Supabase Auth + Storage. However this guide only focus on Supabase db.

We should implement login flow + profile photo update to showcase Auth + Storage. wdyt @hieuwu

Reply:
Actually I plan to create more screens in a separated tab with more action to make use of most APIs interacting with Auth and Storage (resuming, error handling,...). In this first iteration, I focus mostly on the database cause this requires lots of steps and setup. I will introduce Auth + Storage in next iterations. Is this approach okay @phamhieu ?

Reply:
You are right. I will introduce these. It won't take much time actually